### ISSUE TITLE
Batch ChromeOS actions

### ISSUE THREAD

**Issue by jay0lee** [GAM-team/GAM#1668]
Directory API now supports batch actions for Chrome OS devices. See:

https://github.com/jay0lee/google-api-tracker/commit/df330ee989f9a4e0aba7e1277d7d7a8f0e8c6052

All commands should be moved to the batch API and calls to the deprecated API should be removed.

@taers232c fyi

**Comment by taers232c** [comment]
Will do
----
Ross Scroggs
***@***.***

**Comment by taers232c** [comment]
Is this live? The downloaded discovery document doesn't show this nor does this: https://developers.google.com/admin-sdk/directory/reference/rest/v1/chromeosdevices

Thanks
----
Ross Scroggs
***@***.***

**Comment by taers232c** [comment]
OK! The discovery document is updated but its not in the same order as what you posted.
----
Ross Scroggs
***@***.***

**Comment by taers232c** [comment]
These actions seem to have gone away.
pre_provisioned_disable: A pre-provisioned device that cannot enroll. An admin can re-enable the device to allow Zero-Touch Enrollment by changing the status to "pre_provisioned_reenable". Note: From this state the device can be deprovisioned.
pre_provisioned_reenable: Device is pre-provisioned by OEMs. Devices in this state are eligible for Zero-Touch Enrollment.

Can you verify?

Thanks
----
Ross Scroggs
***@***.***

**Comment by taers232c** [comment]
Can you ask your Chrome buddies if there is a limit on the  number of deviceIds in a batch?

Thsnks
----
Ross Scroggs
***@***.***

**Comment by jay0lee** [comment]
Yes but it'll be awhile since the holiday break has started. For now I'd
suggest setting defaults to something very conservative like 10 and we can
adjust from there.

**Comment by jay0lee** [comment]
I think this one is also done @taers232c  ?
